bootstrap/app.php

```diff
@@ -1,17 +1,45 @@
 <?php
 
+use App\Http\Middleware\PreventRequestsDuringMaintenance;
+use App\Http\Middleware\TrimStrings;
+use App\Http\Middleware\TrustProxies;
 use Illuminate\Foundation\Application;
 use Illuminate\Foundation\Configuration\Exceptions;
 use Illuminate\Foundation\Configuration\Middleware;
+use Illuminate\Foundation\Http\Middleware\ConvertEmptyStringsToNull;
+use Illuminate\Foundation\Http\Middleware\ValidatePostSize;
+use Illuminate\Http\Middleware\HandleCors;
+use Illuminate\Routing\Middleware\SubstituteBindings;
+use Illuminate\Support\Facades\Route;
 
 return Application::configure(basePath: dirname(__DIR__))
     ->withRouting(
         web: __DIR__.'/../routes/web.php',
         commands: __DIR__.'/../routes/console.php',
         health: '/up',
+        then: function () {
+            Route::middleware('api')
+                ->prefix('api')
+                ->group(base_path('routes/api.php'));
+
+            Route::middleware('delivery')
+                ->group(base_path('routes/delivery.php'));
+        },
     )
     ->withMiddleware(function (Middleware $middleware) {
-        //
+        $middleware->appendToGroup('delivery', [
+            SubstituteBindings::class
+        ]);
+
+        $middleware->append([
+            // \App\Http\Middleware\TrustHosts::class,
+            TrustProxies::class,
```

Comment:
do we really need to add these here for them to be active? trimstrings sounds pretty basic

Reply:
No, this were duplicates of the default global middlewares. You can check which global middleswares are applied with: `$middleware->getGlobalMiddleware()`, which includes all of the above listed middlewares.